Assignments/todo/MackeyJennifer/todonew2/src/ToDo.js

```diff
@@ -0,0 +1,126 @@
+
+import React, { Component } from 'react';
+
+class ToDo extends Component {
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      edit: false,
+      id: null,
+      fakeData: [{
+        id: '1',
+        title: 'Create To-Do App',
+        isComplete: false,
+      }]
+    }
+  }
+
+  editItem(event) {
+    this.setState({
+      edit: true,
+      id: arguments[0],
+      title: arguments[1]
+    });
+  }
+
+
+  deleteItem() {
+    let id = arguments[0];
```

Comment:
Could simplify this and make it more obvious by:
`deleteItem(id)`

Instead of pulling from arguments